apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx

```diff
@@ -413,6 +415,15 @@ export const DeviceActionDefaultRendering = <R, H extends States, P>({
     });
   }
 
+  if (unresponsive || error instanceof TransportRaceCondition) {
```

Comment:
[Comment] Not relate to your PR but what a generic name for an error `TransportRaceCondition` 😅

Reply:
My thoughts 😅 we should maybe rename it like `TransportPendingOperation`. It would be more clear and less weird